rpc/src/v1/impls/parity.rs

```diff
@@ -200,6 +200,10 @@ impl<C, M, U> Parity for ParityClient<C, M, U> where
 		Ok(self.settings.chain.clone())
 	}
 
+	fn chain_id(&self) -> Result<Option<U256>, Error> {
```

Comment:
1. minor: `U64` would be enough
2. As mentioned earlier `chain_id` is already returned as decimal (`u64`) in transactions and I'd prefer consistency (either all `U64` (hex) or all `u64` (decimal)).

Reply:
@tomusdrw Where's the U64 struct defined?

Reply:
Thought we already have it, but seems we only have U128 and U256. You can create `U64` in `types/uint.rs`.